CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Change Log
+## 2.19.0 - 2025-07-31
```

Comment:
We will likely not meet this release date. Can you make a note on your end to update this date once the release is done.